_posts/2016-01-02-git-and-sensitive-data.markdown

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: "Protect your sensitive Data"
+date: 2016-01-02 13:55:00
+categories: ["open source", "security"]
+author: "schmierkov"
+---
+
+If you are working with open source or if you are going to open source a repository, you should ensure that none of your sensitive data (API Keys, Credentials etc.) can be accessed by anyone.
```

Comment:
@schmierkov Maybe be more specific? "e.g. API Keys, Credentials, Tokens, Passwords"